--- build_tensorflow.sh 2018-12-13 05:31:04.633886152 +0000
+++ build_tensorflow.sh 2018-12-13 05:35:34.299530706 +0000
@@ -60,7 +60,7 @@
     export TF_NEED_OPENCL=0
     export TF_NEED_OPENCL_SYCL=0
     export TF_NEED_CUDA=1
-    export TF_CUDA_VERSION="9.0"
+    export TF_CUDA_VERSION="10.0"
     export CUDA_TOOLKIT_PATH="$(dirname $(dirname $(which nvcc)))"
     export CUDNN_INSTALL_PATH="/usr/lib/aarch64-linux-gnu"
     export TF_CUDNN_VERSION="$(ls -l $CUDNN_INSTALL_PATH | grep -oP '(?<=libcudnn.so.)\s*(\d+)\.(\d*)\.(\d*)\s*' | head -n 1)"
